Nodejs/Product/Nodejs/Commands/DiagnosticsCommand.cs

```diff
@@ -52,137 +58,157 @@ internal sealed class DiagnosticsCommand : Command {
+            var res = new StringBuilder();
             res.AppendLine("Loaded assemblies:");
             foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies().OrderBy(assem => assem.FullName)) {
                 var assemFileVersion = assembly.GetCustomAttributes(typeof(AssemblyFileVersionAttribute), false).OfType<AssemblyFileVersionAttribute>().FirstOrDefault();
 
-                res.AppendLine(string.Format("  {0}, FileVersion={1}",
+                res.AppendLine(Indent(4, string.Format("{0}, FileVersion={1}",
                     assembly.FullName,
-                    assemFileVersion == null ? "(null)" : assemFileVersion.Version
-                ));
+                    assemFileVersion == null ? "(null)" : assemFileVersion.Version)));
             }
-            res.AppendLine();
-            
+            return res.ToString();
+        }
 
+        private static string GetAnalysisLevelInfo() {
+            var res = new StringBuilder();
             res.AppendLine(String.Format("Analysis Level: {0}", NodejsPackage.Instance.IntellisenseOptionsPage.AnalysisLevel.ToString()));
             res.AppendLine();
-            if(NodejsPackage.Instance._analyzer != null) {
+            if (NodejsPackage.Instance._analyzer != null) {
                 var jsAnalyzer = NodejsPackage.Instance._analyzer;
                 res.AppendLine("Default Analysis Log: ");
 
-                using (StringWriter writer = new StringWriter(res)) {
+                using (var writer = new StringWriter(res)) {
                     jsAnalyzer.DumpLog(writer);
                 }
             }
 
-            res.AppendLine(String.Format("IntelliSense Completion Only Tab or Enter to Commit: {0}", NodejsPackage.Instance.IntellisenseOptionsPage.OnlyTabOrEnterToCommit));
-            res.AppendLine();
-
+            res.AppendLine(string.Format("IntelliSense Completion Only Tab or Enter to Commit: {0}", NodejsPackage.Instance.IntellisenseOptionsPage.OnlyTabOrEnterToCommit));
             return res.ToString();
         }
 
-        private static string GetProjectProperty(EnvDTE.Project project, string name) {
-            try {
-                var item = project.Properties.Item(name);
-                if (item != null && item.Value != null) {
-                    return item.Value.ToString();
-                }
-            } catch {
-                // noop
-            }
-            return "<undefined>";
-        }
-
-        public override int CommandId {
-            get { return (int)PkgCmdId.cmdidDiagnostics; }
+        private static string Indent(int count, string text) {
+            var indent = new string(' ', count);
+            var lines = text.Split(new[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
+            var indentedText = lines.Select(line =>
+                string.IsNullOrWhiteSpace(line) ? line : indent + line);
+            return string.Join(Environment.NewLine, indentedText);
```

Comment:
No, all this does it indent every line that is not whitespace. Before, we were just tacking on string with a bunch of spaces in it so multiline text wasn't indented properly.